packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/dialplan.html

```diff
@@ -1,45 +1,48 @@
-<div class="voip-plan__step">
+<div class="group-number__step">
```

Comment:
Could a `group-number__step` be replaced by either a progress tracker (https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-progress-tracker-native--horizontal-default) or a stepper (https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-stepper-introduction--page) ?

Reply:
No it couldn't because, it's not a real step (so no progress tracker or stepper). Perhaps a bad naming if you want.